db/migrate/20230609131618_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :title
+      t.text :text
+      t.integer :likes_counter
+
```

Comment:
[REQUIRED] Kindy include `:comments_counter` to your posts migration as it is a requirement